src/Microsoft.Graph.Cli.Core/IO/NativeResponseProcessor.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Kiota.Abstractions;
+
+namespace Microsoft.Graph.Cli.Core.IO;
+
+public class NativeResponseProcessor : IResponseProcessor
+{
+    public async Task<string?> ExtractStringResponseAsync(IResponseHandler responseHandler)
```

Comment:
This wraps around the http response message type, which @darrelmiller was strongly opposed to when we first started Kiota.
I know I'm late reviewing this, will create a related issue for further discussion